UI/MVUX/MVUX/Presentation/MainPage.xaml

```diff
@@ -0,0 +1,123 @@
+		</Grid.RowDefinitions>
+
+		<Grid.ColumnDefinitions>
+			<ColumnDefinition Width="Auto" />
+			<ColumnDefinition Width="*" />
+			<ColumnDefinition Width="Auto" />
+		</Grid.ColumnDefinitions>
+
+		<Button x:Name="NavViewToggleButton"
+				Grid.Row="0"
+				Grid.Column="0"
+				HorizontalAlignment="Left"
+				Click="NavViewToggleButton_Click"
+				Canvas.ZIndex="10"
+				Margin="10,0,0,0"
+				Style="{StaticResource PaneToggleButtonStyle}" />
+
+		<TextBlock Text="MVUX Samples"
+				   Grid.Row="0"
+				   Grid.Column="1"
+				   HorizontalAlignment="Center"
+				   VerticalAlignment="Center"
+				   FontSize="24"
+				   FontWeight="Bold"
+				   Canvas.ZIndex="10"
+				   Margin="0,0,10,0" />
+
+		<muxc:NavigationView x:Name="NavigationViewControl"
+							 Grid.Row="1"
+							 Grid.ColumnSpan="3"
+							 uen:Region.Attached="True"
+							 IsBackButtonVisible="Collapsed"
+							 IsPaneOpen="True"
+							 IsSettingsVisible="False"
+							 IsPaneToggleButtonVisible="False">
+
+			<muxc:NavigationView.MenuItems>
+
+				<muxc:NavigationViewItem uen:Region.Name="ListFeed"
+										 Content="ListFeed Page">
+					<muxc:NavigationViewItem.Icon>
+						<SymbolIcon Symbol="List" />
+					</muxc:NavigationViewItem.Icon>
+				</muxc:NavigationViewItem>
+
+				<muxc:NavigationViewItem uen:Region.Name="Feed"
+										 Content="Feed Page">
+					<muxc:NavigationViewItem.Icon>
+						<SymbolIcon Symbol="PostUpdate" />
+					</muxc:NavigationViewItem.Icon>
+				</muxc:NavigationViewItem>
+
+				<muxc:NavigationViewItem uen:Region.Name="UpdateState"
+										 Content="How to Update a State">
+					<muxc:NavigationViewItem.Icon>
+						<SymbolIcon Symbol="Refresh" />
+					</muxc:NavigationViewItem.Icon>
+				</muxc:NavigationViewItem>
+
+				<muxc:NavigationViewItem uen:Region.Name="Selection"
+										 Content="Selection Page">
+					<muxc:NavigationViewItem.Icon>
+						<SymbolIcon Symbol="Bullets" />
+					</muxc:NavigationViewItem.Icon>
+				</muxc:NavigationViewItem>
+
+				<muxc:NavigationViewItem uen:Region.Name="Pagination"
+										 Content="Pagination">
+					<muxc:NavigationViewItem.Icon>
+						<SymbolIcon Symbol="Forward" />
+					</muxc:NavigationViewItem.Icon>
+				</muxc:NavigationViewItem>
+
+				<muxc:NavigationViewItem uen:Region.Name="FeedView"
+										 Content="FeedView Page">
+					<muxc:NavigationViewItem.Icon>
+						<SymbolIcon Symbol="View" />
+					</muxc:NavigationViewItem.Icon>
+				</muxc:NavigationViewItem>
+
+				<muxc:NavigationViewItem uen:Region.Name="FeedViewCommand"
+										 Content="FeedView Command">
+					<muxc:NavigationViewItem.Icon>
+						<SymbolIcon Symbol="Play" />
+					</muxc:NavigationViewItem.Icon>
+				</muxc:NavigationViewItem>
+
+				<muxc:NavigationViewItem uen:Region.Name="Messenger"
+										 Content="Update feed using IMessenger">
+					<muxc:NavigationViewItem.Icon>
+						<SymbolIcon Symbol="Play" />
+					</muxc:NavigationViewItem.Icon>
+				</muxc:NavigationViewItem>
```

Comment:
When I select this one nothing really happens, is the region name correct? I don't see any "Messenger" route being defined.